concent-builder/role-tiller.yml

```diff
@@ -0,0 +1,49 @@
+# Tiller permission to all resources in default namespace
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: tiller-manager
+  namespace: default
+rules:
+- apiGroups: ["", "extensions", "apps"]
+  resources: ["*"]
+  verbs: ["*"]
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: tiller-binding
+  namespace: default
+subjects:
+- kind: ServiceAccount
+  name: tiller
+  namespace: tiller
+roleRef:
+  kind: Role
+  name: tiller-manager
+  apiGroup: rbac.authorization.k8s.io
+---
+# Tiller permission to configmaps in namespace when it is deploying, to allow storing release information
```

Comment:
Please reword and put the command near the binding:

`Allow tiller to do anything with config maps in its namespace so that it can store release information`

Reply:
Done.